lib/factory-helper/timestamps.rb

```diff
@@ -0,0 +1,31 @@
+module FactoryHelper
+  class Timestamps < Base
+
+    class << self
+      def between(from = ::Time.at(0), to = ::Time.local(2050, 12, 31,  23,  23,  59, 999_999.999))
```

Comment:
If we have to pick a time, which is unfortunate but due to not being able to overflow Ruby, I would pick the Unix time at 0b01111111_11111111_11111111_11111111_11111111_11111111_11111111_11111111 aka the max signed 64-bit integer